Makefile

```diff
@@ -5,7 +5,7 @@ BIN_NAME=terraform-provider-instaclustr
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
 
 
-VERSION=1.25.0
+VERSION=1.26.0
```

Comment:
Should be a patch version bump as per semver rules as this is pretty much just documentation changes and test fixes, and we aren't adding any new features.